### ISSUE TITLE
Give indepence techs on concede

### ISSUE THREAD

**Issue by MichaelCourtney** [freeorion/freeorion#4511]
Conceding doesn't give the defense boosts of declaring colony independence:

https://freeorion.org/forum/viewtopic.php?t=12773&sid=98e4d04e533c12d096fb5d246c512f32

**Comment by geoffthemedio** [comment]
Not really clear what you want to happen. Should conceding make all the empire's planets effectively declare independence as if they had the building?

**Comment by MichaelCourtney** [comment]
> Not really clear what you want to happen. Should conceding make all the empire's planets effectively declare independence as if they had the building?

Yes, probably the simplest. Though any solution where they get the old defense, shield and troop techs from the old empire is fine.

**Comment by agrrr3** [comment]
IMHO keeping the conceded empire  around with its tech is in the grand scheme of things the lesser trouble. Maybe lets talk about this on next developer meetup?
